### PR TITLE
add user model and migration

### PR DESCRIPTION
**Pull request by HarryPerez**
## Trello Card
https://trello.com/c/XRwp2MQ6/26-modelo-de-usuario

### CODE REVIEW

File: migrations/migrations/20190211153219-create-user.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('users', {
```
Comment (molinajulian) [inline migrations/migrations/20190211153219-create-user.js]: Remove brackets and return

File: app/models/user.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        autoIncrement: true,
+        primaryKey: true,
+        type: DataTypes.INTEGER
+      },
+      firstName: { alowNull: false, type: DataTypes.STRING },
+      lastName: { alowNull: false, type: DataTypes.STRING },
+      email: { alowNull: false, type: DataTypes.STRING, unique: true },
+      password: { alowNull: false, type: DataTypes.STRING }
+    },
+    {}
```
Comment (molinajulian) [inline app/models/user.js]: Add `paranoid:true`. Check this in documentation for models: http://docs.sequelizejs.com/manual/tutorial/models-definition.html

File: app/models/user.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+module.exports = (sequelize, DataTypes) => {
+  const user = sequelize.define(
+    'user',
+    {
+      id: {
```
Comment (molinajulian) [inline app/models/user.js]: Id is  unnecessary in models

File: migrations/migrations/20190211153219-create-user.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        type: Sequelize.STRING
+      },
+      createdAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      },
+      updatedAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      }
```
Comment (molinajulian) [inline migrations/migrations/20190211153219-create-user.js]: Add `deletedAt`

**Review by molinajulian — CHANGES_REQUESTED** [review]

File: app/models/user.js (hunk 2 above)
Comment (HarryPerez) [reply app/models/user.js]: 👍  We don't want to really delete this object.

**Review by molinajulian — APPROVED** [review]

**Pull request merged**